resources/lib/template.py

```diff
@@ -344,7 +344,10 @@ def getProperties( self, elem, items ):
                     continue
                     
                 # We've matched a property :)
-                properties[ name ] = property.text
+                if property.text is not None and property.text != "":
```

Comment:
I'd have to say it's definitely a matter of taste, especially in this case where changing it wouldn't actually simplify the code any (just re-arrange it so the 'false' cases come before the 'true' ones) - and would make it marginally harder to read for me! And when push comes to shove I'm not really a Python guy (indeed, after - according to the commits on this script - 2 years since I first worked with it, there's still things about Python I despise!)

If you feel strongly about it, though, do say and I'll do the re-arrangement :)